cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -63,7 +83,7 @@ <h4 class="u-mb5">
             {{ post.text | safe }}
         </p>
         {% endif %}
-        <p class="date">
+        <p class="date u-mt10 u-mb0">
```

Comment:
That won't do what you're expecting, the margins [come directly from](https://github.com/cfpb/cf-typography/blob/gh-pages/src/cf-typography.less#L236) `.date` styles